paddle/cuda/src/hl_warpctc_wrap.cc

```diff
@@ -89,7 +89,8 @@ void hl_warpctc_init(const size_t blank,
   if (useGpu) {
 #ifdef __HIPCC__
     options->loc = CTC_GPU;
-    options->stream = STREAM_DEFAULT;
+    //options->stream = STREAM_DEFAULT;
```

Comment:
why do we disable it here? If it has to be, we should leave a "XXX" flag so later we can re-visit.